prov/gni/test/cntr.c

```diff
@@ -98,6 +98,7 @@ static inline void cntr_setup_eps(void)
 	hints->mode = ~0;
 
 	hints->fabric_attr->name = strdup("gni");
+	hints->caps = GNIX_EP_RDM_PRIMARY_CAPS;
```

Comment:
Is this needed for this test to work properly?  If not, as I mentioned in #866, I think we should just leave the caps alone.